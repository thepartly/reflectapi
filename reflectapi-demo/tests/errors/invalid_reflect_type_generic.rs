#[derive(reflectapi::Input, reflectapi::Output)]
struct MyStruct<T> {
    field: u8,
}

fn main() {}
