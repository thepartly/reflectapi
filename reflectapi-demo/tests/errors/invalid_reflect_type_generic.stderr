error[E0392]: type parameter `T` is never used
 --> tests/errors/invalid_reflect_type_generic.rs:2:17
  |
2 | struct MyStruct<T> {
  |                 ^ unused type parameter
  |
  = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`
  = help: if you intended `T` to be a const parameter, use `const T: /* Type */` instead
